src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -627,18 +626,17 @@ public boolean processInvalidUrl(final WebView view, String url) {
                         mRedirectUrl));
                 view.stopLoading();
                 return true;
-            }
-            
-            //check if the redirect URL is under SSL protected
-            if(!url.toLowerCase(Locale.US).startsWith(AuthenticationConstants.Broker.REDIRECT_SSL_PREFIX))
-            {
+            } else if (url.startsWith("about:blank")) {
+                Logger.v(TAG, "It is an blank page request");
+                return true;
+            } else if(!url.toLowerCase(Locale.US).startsWith(AuthenticationConstants.Broker.REDIRECT_SSL_PREFIX)){
```

Comment:
The "about:blank" is treated as invalid redirect URL here. The function processInvalidUrl returns true if the URL is invalid. 
....
else if (url.startsWith("about:blank")) {  
                Logger.v(TAG, "It is an blank page request");  
               return true;  
} ....